randomwallpaper@iflow.space/schemas/org.gnome.shell.extensions.space.iflow.randomwallpaper.gschema.xml

```diff
@@ -2,20 +2,20 @@
 <schemalist gettext-domain='gnome-shell-extensions'>
 
     <enum id='org.gnome.shell.extensions.space.iflow.randomwallpaper.sources'>
-        <value value='0' nick='unsplash'/>
-        <value value='1' nick='wallhaven'/>
-        <value value='2' nick='reddit'/>
-        <value value='3' nick='genericJSON'/>
+        <value value='0' nick='unsplash' />
+        <value value='1' nick='wallhaven' />
+        <value value='2' nick='reddit' />
+        <value value='3' nick='genericJSON' />
```

Comment:
Again sorry, that was the auto-formatter of VSCode kicking in.

> it would be good not to touch these things if they are not related to the subject of your commit.

Gotcha! But this will show up again in the next PRs.

Reply:
Yeah, no problem! As I said, I haven't been very careful about this in this repository either.